templates/home.html

```diff
@@ -38,8 +37,27 @@
         </div>
     </div>
 </header>
-<main>
-
-</main>
+    
+<div class="tna-page">
+  <div class="container">
+    <div class="row">
+      <main class="col-xs-12 col-sm-12" role="main">
+        <article>
+          <div class="tna-page__header">
+            <h1>Would you like help choosing your seat in the Document Reading Room(DRR)?</h1>
+          </div>
+          <div class="tna-page__entry">
+            <ul class="tna-ul--no-bullet">
+                <li><a href="{{ url_for('help') }}" class="tna-button tna-button--primary" role="button">Yes</a></li>
```

Comment:
These links should not have `role="button"` on them because it is misleading for users of assistive technologies. Please remove this from all buttons in this app.